src/tokens/eth/0x30fef258d2728f9d1edf038059c725faf785697e.json

```diff
@@ -0,0 +1,31 @@
+{
+  "symbol": "PESO",
+  "name": "PESOTOKEN",
+  "type": "ERC20",
+  "address": "0x30fef258d2728f9d1edf038059c725faf785697e",
+  "ens_address": "",
+  "decimals": 2,
+  "website": "http://pesoexchange.com",
+  "logo": {
+    "src": "https://res.cloudinary.com/cloudimgstorage/image/upload/v1560173317/PESO-LOGO.png",
```

Comment:
New Logo Need To Be Updated With This One: https://res.cloudinary.com/cloudimgstorage/image/upload/v1560754489/PESO-LOGO-UPDATED-V1.png